webapp/src/shared/components/Input/EditUserForm.tsx

```diff
@@ -53,6 +57,16 @@ export default function EditUserForm({
         onSubmit={handleOnSubmit}
       >
         <div className="inputs-container">
+          <Input
```

Comment:
comment (non blocking):
We have to rethink the UI.

Reply:
If we like how the form looks with this additional input, I could update Figma. If not, please suggest what we can improve.

Reply:
Please, take a look at the ongoing discussions in figma/slack on the topic, and feel free to share your own ideas on the topic.